kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -5932,7 +5936,14 @@ public void commitResource(ContentResourceEdit edit, int priority) throws OverQu
                 //This might not want to be set as it would advise the detector
                 metadata.set(Metadata.RESOURCE_NAME_KEY, edit.getId());
                 metadata.set(Metadata.CONTENT_TYPE, currentContentType);
-                String newmatch = DETECTOR.detect(TikaInputStream.get(buff), metadata).toString();
+                String newmatch = "";
+                //If we are ignoring the content for this extension, don't give it any data
+                if (m_ignoreContent != null && Arrays.asList(m_ignoreContent).contains(FilenameUtils.getExtension(edit.getId()))) {
```

Comment:
@buckett Yeah, a String aray is what's returned from SCS.getStrings, and this is only used once,  so it seemed about the same in either place. I debated just reading it once but it seems like the pattern of always reading it from SCS is better for a possible future of dynamic properties. I think it would have been generally more convenient if SCS returned a List instead.